Qonto/Model/TransactionFilterBuilder.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace neyric\Qonto\Model;
+
+use Cassandra\Date;
```

Comment:
Is this used somewhere ?

Reply:
Yes for validating date parameter in filtering (ISO8601) in TransactionFilterBuilder.php